src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border-radius: $cardsBorderRadius;
+  border: $cardsBorder;
+  display: flex;
+  flex-direction: column;
+
+  background: $white-color;
```

Comment:
it is white by default but anyway 😄

Reply:
I'm deleted it. Or should they be moved to index.scss?

Reply:
Deleted 'background-color: $white-color;' from star's block too